### ISSUE TITLE
broadcast isn't implemented on backend [xpu]

### ISSUE THREAD

**Issue by Peach-He** [intel/torch-ccl#33]
Hi,
When I try to use torch-ccl + torch DDP + ipex, there is a runtime error called "broadcast isn't implemented on backend [xpu]", any suggestions on this issue?
Test code:
```
import os
import torch
import torch.nn as nn
from torch.nn.parallel import DistributedDataParallel as DDP
import torch.distributed as dist
import torch_ccl
import torch_ipex as ipex

class Model(nn.Module):
    def __init__(self):
        super(Model, self).__init__()
        self.linear = nn.Linear(4, 5)

    def forward(self, input):
        return self.linear(input)


if __name__ == "__main__":
    
    os.environ['RANK'] = os.environ.get('PMI_RANK', -1)
    os.environ['WORLD_SIZE'] = os.environ.get('PMI_SIZE', -1)
  
    # Initialize the process group with ccl backend
    dist.init_process_group(backend='ccl')

    model = Model().to(ipex.DEVICE)
    if dist.get_world_size() > 1:
        model=DDP(model)

    for i in range(3):
        input = torch.randn(2, 4)
        labels = torch.randn(2, 5)
        loss_fn = nn.MSELoss()
        optimizer = torch.optim.SGD(model.parameters(), lr=0.001)

        # forward
        res = model(input.to(ipex.DEVICE))
        L=loss_fn(res, labels.to(ipex.DEVICE))

        # backward
        L.backward()

        # update
        optimizer.step()
```
Error:
```
[0] Traceback (most recent call last):
[0]   File "test_ddp.py", line 28, in <module>
[0]     model=DDP(model)
[0]   File "/opt/intel/oneapi/intelpython/latest/envs/rnnt/lib/python3.7/site-packages/torch/nn/parallel/distributed.py", line 496, in __init__
[0]     dist._verify_model_across_ranks(self.process_group, parameters)
[0] RuntimeError: torch_ccl: broadcast isn't implementd on backend [xpu].
```
SW version:
```
pytorch 1.9, ipex 1.9, torch-ccl 1.9
```

**Comment by chengjunlu** [comment]
Hi @Peach-He,

1. The import sequence of the torch-ccl and the ipex matters for XPU device. The ipex should be imported before the torch-ccl.
2. The heterogeneous computing device (XPU) has not been released for the pytorch 1.9. Where did you get the release?

**Comment by Peach-He** [comment]
Fixed by upgrate to version 1.10
